labels.json

```diff
@@ -0,0 +1,39 @@
+
+[
+  { "name": "Package: analyzer", "color": "#f9df95" },
+  { "name": "Package: build", "color": "#f9df95" },
+  { "name": "Package: bundler", "color": "#f9df95" },
+  { "name": "Package: cli", "color": "#f9df95" },
+  { "name": "Package: project-config", "color": "#f9df95" },
+  { "name": "Package: linter", "color": "#f9df95" },
+  { "name": "Browser: Chrome", "color": "#3477c9" },
```

Comment:
Do we have enough browser or os-specific bugs to be worth making browser and os-specific labels?

Reply:
For platform, and in the CLI we definitely have, especially when we were working on initial Windows support. In other repos, the need has occasionally been there, mainly for windows. For Browser we've had lots of browser specific issues.